lexos/managers/utility.py

```diff
@@ -287,11 +287,11 @@ def generate_statistics(file_manager: FileManager) -> \
         cull=culling)
 
     file_info_list = []
-    for count, label in enumerate(labels):
+    for count, label in enumerate(dtm_data.index.values):
         file_info_list.append(information.FileInformation(
-            count_list=count_matrix[count, :], file_name=label))
-    corpus_info = information.CorpusInformation(count_matrix=count_matrix,
-                                                labels=labels)
+            count_list=dtm_data.values[count, :], file_name=label))
```

Comment:
I think the way to get the data is `dtm_data.data` not `dtm_data.values` but I could be wrong. 
Please double check.

This for loop is getting a little out of hand, find a way to make it pretty.